meta-resin-morty/recipes-connectivity/dnsmasq/dnsmasq_%.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI_remove = "file://0001-Treat-REFUSED-not-SERVFAIL-as-an-unsuccessful-upstre.patch"
```

Comment:
I would prefer to remove it from common and:
1. keep in common the patch (to avoid duplication)
2. all non morty have SRC_URI = "file://0001-Treat-REFUSED-not-SERVFAIL-as-an-unsuccessful-upstre.patch"